src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -531,6 +539,36 @@ public string DictionaryLink
                 }
                 finalLink += this.ShortenName;
 
+                // Check if the method has overloads
+                libraryServices = DynamoModel.StaticLibraryServices;
```

Comment:
@yeexinc I would not suggest adding a new static property for `LibraryServices`. Could you try accessing it in the following way:
```
 var dynView = nodeHelpPrompt.Owner as DynamoView;
 var dynViewModel = dynView.DataContext as DynamoViewModel;
 dynViewModel.Model.LibraryServices.ConstructUrl();
```